blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,275 @@
+a corporate version and an individual version. If you are contributing as an employee of a company, you will need to sign the corporate version, otherwise the individual version
+is the right version for you! To sign the CLA, all you need to do is follow the steps in the comment posted to your PR by the CLA bot.
+
+#### Interacting with Prow
+
+[Prow](https://docs.prow.k8s.io/docs/){:target="_blank"} is a Kubernetes-based bot which is used in many CNCF projects to manage issues and PRs. In Knative, it is generally Prow which merges the PRs - not other contributors or maintainers!
+To determine which PRs to merge, Prow is given sets of criteria. These are generally a set of labels, as well as that the required tests all passed. In Knative, the two labels
+which your PR needs in order to get merged are "lgtm" and "approved". Anyone can add the "lgtm" by commenting "/lgtm" on a PR, but only approvers can add the "approved" label.
+Additionally, if you have a "do-not-merge/hold" (which you add by commenting "/hold"), or a "do-not-merge/work-in-progess" label (which you get by making a draft PR), the bot
+will not merge your changes. 
+
+In order for tests to pass on your PR, they will need to be run. By default, however, they are not run! In order for them to run, a member of the knative or knative-sandbox organization
+will have to comment "/ok-to-test" (which organization they must be a member of depends on which organization your repository is under). The "/ok-to-test" command adds the "ok-to-test"
+label to your PR. Generally speaking, whoever reviews your PR will do this, but if they haven't you can always ask them why they haven't. If you join the knative or knative-sandbox
+organizations, you will no longer need to have the ok-to-test label applied to your PRs - prow will run the tests automatically! So, once you have made a few contributions,
+join the organizations!
+
+As we mentioned earlier, to assign an issue to yourself, you can comment "/assign", and Prow will assign the issue to you. You can also do "/assign @username", and Prow will assign
+the issue to the user with username. You can also use the command with PRs to assign them to specific users.
+
+Something you will probably want to do on your PR is re run the tests if one of them fails and you think it was a flake. To do this, just comment "/retest" to rerun all tests,
+or "/retest-required" to just retest all failed and required tests.
+
+One last command that you will find useful is "/cc @username", which requests a review from the user with username. You can put multiple users into this command and request
+more than one review in one go (e.g. "/cc @user1 @user2").
+
+To find out about all the commands that Knative's instance of Prow supports, as well as their usage, you can read the [Prow commands page](https://prow.knative.dev/command-help){:target="_blank"}.
+
+![](/blog/images/getting-started-blog-series/post2/doodle003.png)
```

Comment:
Would you be interested in mentioning how the review process looks like?
- Who does the review?
- How long it typically takes?
- Should the contributor take the review comments personal?
- Use the learning opportunity from maintainers who are more experienced
- ...

Reply:
I think we covered that in the previous blog post, should we link back to that one here?

Reply:
It is fine then, no need for duplication 👍